README.md

```diff
@@ -12,8 +12,6 @@
 	<br />
 	<p>
 		<a href="https://discord.gg/E3z8MDnTWn"><img src="https://img.shields.io/discord/901426442242498650?color=5865F2&logo=discord&logoColor=white" alt="Disploy's Discord server" /></a>
-		<a href="https://github.com/disploy/Disploy/actions"><img src="https://github.com/Disploy/disploy/actions/workflows/tests.yml/badge.svg" alt="Tests status" /></a>
-	</p>
```

Comment:
You accidentally deleted the closing tag for the paragraph.

Reply:
Fixed in the new PR